composer.json

```diff
@@ -15,8 +15,8 @@
     "prefer-stable": true,
     "require": {
         "php": "^7.3 || ^8",
-        "phpspec/prophecy": "^1.3",
-        "phpunit/phpunit":"^9.1"
+        "phpspec/prophecy": "^1.18 || dev-master as 1.18",
```

Comment:
this should be removed. The branch-alias in prophecy means that the master branch satisfies a `^1.18` constraint anyway.

Reply:
It's already pointed [above](https://github.com/phpspec/prophecy-phpunit/pull/45/files/1fd9d97a718c188e5b2ae2609e1f52def157b5b7#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34).